tools/topology/topology1/platform/intel/intel-generic-dmic-rtnr.m4

```diff
@@ -0,0 +1,135 @@
+#
```

Comment:
Can you please comment why a different version of this macro is needed. Could more flexibility added to intel-generic-dmic.m4 meet your needs. Diff of the the two files that there's changes but more like hard coding parameters instead of using the definable macros.

Reply:
The topology is modified which uses intel-generic-dmic.m4 instead. Thanks!